src/test/java/pathfinding/ConcreteGraphTest.java

```diff
@@ -74,6 +69,44 @@ public void findPath() throws Exception
 		ArrayList<Node> orderedSolution = graph.findPath(straightNodes[0], straightNodes[straightNodes.length - 1]);
 		assertNotNull(orderedSolution);
 		for (int i = 0; i < 100; i++)
-			assertEquals(orderedSolution.get(i), straightNodes[99-i]);
+			assertEquals(orderedSolution.get(i), straightNodes[99 - i]);
+	}
+
+	@Test
+	public void textDirect() {
```

Comment:
The function in ConcreteGraph that returns a list of directions needs a better name, something that could tell you approximately what it does without having to check the javadocs. Maybe something like `getPathDirections()` ?